test/test.config.js

```diff
@@ -197,7 +197,7 @@ describe('Config API', function() {
                     });
                 });
             } else if (localforage.driver() === localforage.LOCALSTORAGE) {
-                var dbValue = JSON.parse(localStorage['My Cool App/some key']);
+                var dbValue = JSON.parse(localStorage['My Cool App/storeFront/some key']);
```

Comment:
Not my choice of name. :-)

I've just adapted the test to make it pass with the implementation of the `storeName` namespace of this PR. The original name was here:

https://github.com/sebweaver/localForage/blob/issue-342-multiple-stores-on-same-db/test/test.config.js#L78